proto/src/determined/api/v1/config_policies.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package determined.api.v1;
+option go_package = "github.com/determined-ai/determined/proto/pkg/apiv1";
+
+import "google/protobuf/struct.proto";
+
+// PutWorkspaceConfigPoliciesRequest is a complete replacement of config
+// policies for the workspace and workload type.
```

Comment:
// PutWorkspaceConfigPoliciesRequest is a complete replacement of a workspace's config policies (its invariant config and constraints) for a given workload type.

Reply:
Also, are we planning to create the specified config policies if none exist? If so, we should probably change "is a complete replacement of" to "sets"